src/rules/no-flowmax-in-forwardref.coffee

```diff
@@ -0,0 +1,23 @@
+module.exports =
+  meta:
+    docs:
+      description: 'Flag uses of flowMax() within forwardRef()'
+      category: 'Possible Errors'
+      recommended: yes
+    schema: [
+      type: 'object'
+      additionalProperties: no
+    ]
+    fixable: 'code'
```

Comment:
This rule doesn't support fixing (there's no practical way to correctly refactor it using ESLint fixing), so remove `fixable: 'code'`